web-client/src/components/PopUpForgotPassword.jsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import * as yup from "yup";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import Dialog from "@material-ui/core/Dialog";
+
+const styles = makeStyles({
+  main: {
+    display: "inline-block",
+  },
+  content: {
+    padding: "30px 100px",
+  },
+  form: {
+    display: "flex",
+    flexDirection: "column",
+    float: "left",
+    textAlign: "left",
+    maxWidth: "400px",
+    margin: "16px",
+  },
+  textField: {
+    marginBottom: "32px",
+    width: "300px",
+  },
+  send: {
+    width: "100px",
+    textTransform: "none",
+    float: "right",
+    marginLeft: "auto",
+    display: "block",
+  },
+  textButton: {
+    textTransform: "none",
+  },
+  message: {
+    textAlign: "center",
+    marginTop: "30px",
+    marginBottom: "30px",
+  },
+});
+
+const sendSuccessfullyStyle = {
+  visibility: "visible",
+  color: "green",
+};
+const sendUnsuccessfullyStyle = {
+  visibility: "visible",
+  color: "red",
+};
+const defaultStyle = {
+  visibility: "hidden",
+};
+
+export default function PopUpForgotPassword({SET_EMAIL}) {
```

Comment:
I think it's better to use here `props`, instead `{SET_EMAIL}`. It'll work in the same way, moreover you used this approach/notation in SignInForm

Reply:
Ok